app/dialog/otioproperties/otiopropertiesdialog.h

```diff
@@ -0,0 +1,39 @@
+
+#ifndef OTIOPROPERTIESDIALOG_H
+#define OTIOPROPERTIESDIALOG_H
+
+#include <QDialog>
+#include <QTreeWidget>
+
+#include "common/define.h"
+#include "opentimelineio/timeline.h"
+#include "node/project/sequence/sequence.h"
+#include "node/project/project.h"
+
+namespace olive {
+
+  /**
+   * @brief Dialog to load setting for OTIO sequences.
+   *
+   * Takes a list of Sequences and allows the setting of options for each.
+   */
+  class OTIOPropertiesDialog : public QDialog {
+    Q_OBJECT
+   public:
+    OTIOPropertiesDialog(QList<Sequence*> sequences, Project* active_project, QWidget* parent = nullptr);
```

Comment:
Might be nice to change this to `const QList<Sequence*> &sequences` to try saving a deep copy.